go.mod

```diff
@@ -69,7 +69,7 @@ require (
 	google.golang.org/grpc v1.44.0
 	google.golang.org/protobuf v1.27.1
 	gopkg.in/square/go-jose.v2 v2.6.0
-	gotest.tools v2.2.0+incompatible
+	gotest.tools v2.2.0+incompatible // indirect
```

Comment:
We should probably tidy this so it ends up in the 2nd require section.

Reply:
Is this a convention we are enforcing by hand? If not, I think something is busted with our `make tidy` command because it doesn't do this automatically.

Reply:
That is the convention used by the go tool. If you have two sections already, it won't shuffle a dependency from one to the other, but when adding deps it will but direct in the first and indirect in the second. If you remove the indirect dep from the first and run `tidy`, it will show up again in the second.

Reply:
I see, thanks for clarifying.

On a related note - I'm not sure why, but every time I add a new dependency, `make tidy` wants to put it in a new, third `require` block in-between the direct and indirect dependency blocks. Did something change recently in the Go tooling?